### ISSUE TITLE
Flows/Text Template: More examples 

### ISSUE THREAD

**Issue by kris-watts-gravwell** [gravwell/wiki#579]
## Instructions
Labels for change type and priority are automatically assigned at the time of creation. 
**The default priority is Low. Please change the priority label if this requires more attention.**

Here are suggestions to help you set the correct priority but changes can be made at your discretion.

If this request is related to:  
  - Current series objectives
  - Next major release objectives
  - Impacting or Requested by multiple customers
please set the priority to High.

If this request does not meet the above criteria but is more important,
please set the priority to Medium. 

## What is the enhancement to be made?

Text Templates/Mustache templates are just **HARD**.  We should add a bunch more examples on how to use them, how to get at them, how to get at search data/fields/evs.

Basically a whole big extra section on how to.

## Why should we make this change? (Business justification? What problem is the feature trying to solve?)

We get tons of questions, we should lay out several of the basic use cases very clearly with direct examples.

Examples like:

- format email body
- get some EVs out of a search and put in email subject
- webhook body with some formatted values

## Any other comments?

Basically sweep your current usage and add them as examples, lots and lots of examples.
